src/compile.js

```diff
@@ -525,6 +525,15 @@ Compiler.prototype.ccall = function (e) {
     }
     else {
         out ("$ret;"); // This forces a failure if $ret isn't defined
+        if (Sk.python3 && e.func.id && e.func.id.v === "super" && args.length == 0) {
```

Comment:
I couldn't figure out how python3 got that `__class__` variable, so I figured I'd venture out and ask a stack overflow question see: https://stackoverflow.com/a/45522873/117242 

I'm now trying to figure out how to walk the ast from `cclass` to annotate it where we need the `__class__` variable inserted into the scope of the function call.